spec/use_cases/reservations/cancel_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Reservations::UseCases::Cancel do
+  describe ".call" do
+    let(:seats) { [10, 11, 12, 13] }
```

Comment:
you can generate array of a sequence of numbers like so:
```suggestion
    let(:seats) { [*10..13] }
```